Mathlib/Control/Combinators.lean

```diff
@@ -29,38 +42,58 @@ export List (mapM mapM' filterM foldlM)
 
 namespace Monad
 
+@[inherit_doc List.mapM]
 def mapM :=
   @List.mapM
 
+@[inherit_doc List.mapM']
 def mapM' :=
   @List.mapM'
 
+@[inherit_doc joinM]
 def join :=
   @joinM
 
+@[inherit_doc List.filterM]
 def filter :=
   @filterM
 
+@[inherit_doc List.foldlM]
 def foldl :=
   @List.foldlM
 
+@[inherit_doc condM]
 def cond :=
   @condM
```

Comment:
These seem insane? Why are we not deleting them, rather than copying doc-string to them?

Reply:
#23753

Reply:
@BoltonBailey maybe you could merge master and ping me to approve anything remaining in this PR :)